.taskcluster.yml

```diff
@@ -0,0 +1,321 @@
+---
+version: 1
+reporting: checks-v1
+policy:
+    pullRequests: collaborators
+tasks:
+    - $let:
+          # XXX Set to `true` for private repos
+          privateRepo: false
+          # XXX Use
+          # `system` for system add-on,
+          # `privileged` for AMO or self-hosted privileged add-on,
+          # `mozillaonline-privileged` for Mozilla China add-on,
+          # `normandy-privileged` for normandy add-on
+          # to enable siging on push/PR.
+          xpiSigningType: ""
```

Comment:
We should probably set a type here, although I am not 100% clear which one. Presumably `normandy-privileged` or `system`?
Do we know how to distribute the add-on yet?

Reply:
Quoting the instructions linked in the PR description:

> To enable signing on push, set the xpiSigningType in .taskcluster.yml to the appropriate addon type.

My understanding is that we can set `xpiSigningType` later, once we know which type is actually is and we want signing.